sale_cancel_reason/data/sale_order_cancel_reason.xml

```diff
@@ -1,14 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data noupdate="1">
-        <record id="cancel_reason_only_quotation" model="sale.order.cancel.reason">
-            <field name="name">Just for Quotation</field>
-        </record>
-        <record id="cancel_reason_no_service_needed" model="sale.order.cancel.reason">
-            <field name="name">Service no longer needed</field>
-        </record>
-        <record id="cancel_reason_other_provider" model="sale.order.cancel.reason">
-            <field name="name">Other Service Provider selected</field>
-        </record>
-    </data>
-</openerp>
+<data noupdate="1">
```

Comment:
sorry if it seems like a silly question.  Does it not require <odoo> tag in 9.0?